config/initializers/simple_form.rb

```diff
@@ -27,7 +27,7 @@
     b.optional :min_max
     b.optional :readonly
     b.use :label, class: 'text-bold'
-    b.use :hint,  wrap_with: { tag: 'div', class: 'usa-hint' }
+    b.use :hint,  wrap_with: { tag: 'label', class: 'usa-hints' }
```

Comment:
I'm not familiar with the `usa-hints` class. Is it provided by the design system?